core/XH.ts

```diff
@@ -392,18 +392,28 @@ export class XHApi {
     /**
      * Trigger a full reload of the current application.
      *
-     * @param path - relative path to reload (e.g. 'mobile/').  Defaults to the
-     * existing location pathname.
+     * @param opts - Options for the reload. Can be either:
```

Comment:
This is good, but I think take it all way to export an explicit type `ReloadAppOptions`  that is  `string | {path?: string; removeQueryParams?: boolean}`.

This is just easier to document in tsdoc and looks better in the signature